### PR TITLE
can_manage_metadata and categories on motion create

### PR DESCRIPTION
**Pull request by FinnStutzenstein**
- Right permissions for edit and create
- enable categories to be always whitelisted
- fixed bug in search-value-selector

### CODE REVIEW

File: client/src/app/shared/components/search-value-selector/search-value-selector.component.ts [hunk 1]
```diff
@@ -59,10 +59,30 @@ export class SearchValueSelectorComponent implements OnInit {
...
+        this._inputListSubject = value;
+        this._inputListSubscription = this._inputListSubject.subscribe(values => {
+            this.filterItems();
+        });
+    }
+
+    /**
+     * The inputlist subject.
+     */
+    private _inputListSubject: BehaviorSubject<Selectable[]>;
```
Comment (jsaalfeld) [inline client/src/app/shared/components/search-value-selector/search-value-selector.component.ts]: Variable declaration should be on top.

File: client/src/app/shared/components/search-value-selector/search-value-selector.component.ts [hunk 2]
```diff
@@ -59,10 +59,30 @@ export class SearchValueSelectorComponent implements OnInit {
...
+
+    /**
+     * Saves the current subscription to _inputListSubject.
+     */
+    private _inputListSubscription: Subscription = null;
```
Comment (jsaalfeld) [inline client/src/app/shared/components/search-value-selector/search-value-selector.component.ts]: Variable declaration should be on top.

File: client/src/app/shared/directives/perms.directive.ts [hunk 3]
```diff
@@ -39,6 +39,8 @@ export class PermsDirective extends OpenSlidesComponent {
      */
     private alternative: boolean;
 
+    private complement: boolean;
```
Comment (jsaalfeld) [inline client/src/app/shared/directives/perms.directive.ts]: Add documentation for variable

File: client/src/app/shared/directives/perms.directive.ts [hunk 4]
```diff
@@ -85,6 +87,12 @@ export class PermsDirective extends OpenSlidesComponent {
         this.updateView();
     }
 
+    @Input('osPermsComplement')
```
Comment (jsaalfeld) [inline client/src/app/shared/directives/perms.directive.ts]: Documentation is missing for the function

File: client/src/app/site/motions/components/motion-detail/motion-detail.component.ts [hunk 5]
```diff
@@ -275,6 +276,8 @@ export class MotionDetailComponent extends BaseComponent implements OnInit {
     public async saveMotion(): Promise<void> {
         const newMotionValues = { ...this.metaInfoForm.value, ...this.contentForm.value };
 
+        console.log('save:', newMotionValues);
```
Comment (jsaalfeld) [inline client/src/app/site/motions/components/motion-detail/motion-detail.component.ts]: Remove `console.log`

File: openslides/motions/views.py [hunk 6]
```diff
@@ -134,14 +134,15 @@ def create(self, request, *args, **kwargs):
                 'title',
                 'text',
                 'reason',
+                'category_id',
```
Comment (normanjaeckel) [inline openslides/motions/views.py]: Are you that you want to allow users without managing perms to set the category of the new motion?

**Comment by FinnStutzenstein** [comment]
@emanuelschuetze ping..

**Comment by jsaalfeld** [comment]
As @emanuelschuetze wrote above:

> Ok, as discussed.

@normanjaeckel  had the exact same question I had (Thats why I pinged him in my original question ;))

File: openslides/motions/views.py (hunk 6 above)
Comment (emanuelschuetze) [reply openslides/motions/views.py]: Yes, I'm sure. It's a valid use case. We already discussed it on last code sprint meeting. - Maybe we can add a config later.

**Pull request merged**